src/lasp_default_broadcast_distribution_backend.erl

```diff
@@ -1150,6 +1143,36 @@ schedule_delta_garbage_collection() ->
         state_based ->
             ok
     end.
+%% @private
+schedule_plumtree_memory_report() ->
+    case lasp_config:get(memory_report, false) of
+        true ->
+            erlang:send_after(?PLUMTREE_MEMORY_INTERVAL, self(), plumtree_memory_report);
+        false ->
+            ok
+    end.
+
+%% @private
+schedule_memory_utilization_report() ->
+    case lasp_config:get(instrumentation, false) of
+        true ->
+            erlang:send_after(?MEMORY_UTILIZATION_INTERVAL, self(), memory_utilization_report);
+        false ->
+            ok
+    end.
+
+%% @private
+plumtree_memory_report() ->
+    PlumtreeBroadcast = erlang:whereis(plumtree_broadcast),
+    lager:info("Plumtree message queue: ~p",
+               [process_info(PlumtreeBroadcast, message_queue_len)]),
+    lager:info("Our message queue: ~p",
+               [process_info(self(), message_queue_len)]).
+
+%% @private
+memory_utilization_report() ->
+    TotalBytes = orddict:fetch(total, erlang:memory()),
```

Comment:
> The first element Type is an atom describing memory type. The second element Size is the memory size in bytes.

http://erlang.org/doc/man/erlang.html#memory-0

I was assuming the transmission is the same
because I just reused the function that gives megabytes